content/doc/book/getting-started/installing.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: section
+---
+:doctitle: Installing Jenkins
+:notitle:
+:description:
+:author: Liam Newman
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+
+[NOTE]
+====
+This is still very much a work in progress
+====
+
+[IMPORTANT]
+====
+This section is part of _Getting Started_.
+It provides a golden path for a *basic* jenkins configuration on a number of platforms.
+It *DOES NOT* cover the full range of considerations or options for installing Jenkinks.
+See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
+====
+
+== Overview
+
+
+== Pre-install
+
+=== System Requirements
+
+WARNING: These are *starting points*.  For a full discussion of factors see: link:/doc/book/hardware-recommendations/[Discussion of hardware recommendations]
+
+Minimum configuration:
+
+* Java 7
+* 512MB free memory
+* 5GB free disk space
+
+Recommended Initial Configuration:
+
+* Java 8
+* 1GB free memory
+* 50GB free disk space
+
+=== Experimentation, Staging, or Production?
+
+How you configure Jenkins will differ significantly depending on
+
+See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
+
+=== Stand-alone or Servlet?
+
+See link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
+
+
+== Installation
+
+[WARNING]
+====
+These are *clean install* instructions for *non-Production* environments.
+If you have a Jenkins server already running on a system and want to upgrade, see link:/doc/book/getting-started/upgrading/[Upgrading Jenkins].
+If you are setting up a production Jenkins server, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation].
+====
+
+=== Unix/Linux
+
+==== Debian/Ubuntu
+On Debian-based distributions, such as Ubuntu, you can install Jenkins through `apt-get`.
+
+Recent versions are available in link:http://jenkins-ci.org/debian/[an apt repository]. Older but stable LTS versions are in link:http://pkg.jenkins-ci.org/debian-stable/[this apt repository].
+
+You need to have a JDK and JRE installed. openjdk-7-jre and openjdk-7-jdk are suggested.
+
+WARNING: GCJ is known to be problematic. It is used on ubuntu systems.  - see link:https://issues.jenkins-ci.org/browse/JENKINS-743[JENKINS-743].
+
+[source,bash]
+----
+wget -q -O - https://jenkins-ci.org/debian/jenkins-ci.org.key | sudo apt-key add -
```

Comment:
I would prefer this say `wget -q -O - https://pkg.jenkins.io/debian/jenkins.io.key | sudo apt-key add -`


(there's a [stupid VirtualHost redirect rule](https://github.com/jenkins-infra/jenkins-infra/blob/staging/dist/profile/manifests/catchall.pp#L65) to make that previous URL work, and it annoys me greatly to have to carry it forward)